src/autotype/AutoTypeSelectDialog.cpp

```diff
@@ -74,7 +75,7 @@ AutoTypeSelectDialog::AutoTypeSelectDialog(QWidget* parent)
             m_ui->search->setFocus();
         } else {
             // Reset to original match list
-            m_ui->view->setMatchList(m_matches, true);
+            m_ui->view->setMatchList(m_matches);
```

Comment:
Should we add a `selectFirstMatch` call here?

Reply:
I haven't seen any change in the behavior but for completeness why not.

Reply:
Added the call.